doc/ref/configuration/minion.rst

```diff
@@ -658,15 +658,12 @@ Examples:
    - FQDN = king_bob.bar.org --> minion_id = king_bob
 
 
-For more information, please see `Issue #49212`_ and `Pull Request #49378`_.
+For more information, please see :pull:`49378`.
```

Comment:
Why was the issue number removed here, instead of using the `:issue:` RST role?

Reply:
For no good reason. I just commited the `:issue:` RST role